README.md

```diff
@@ -1 +1,260 @@
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
+
+OPTIONAL SECTIONS:
+- FAQ
+
+After you're finished please remove all the comments and instructions!
+-->
+
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  
+  <h3><b>Microverse README Template</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [My-Portfolio_Mobile-Version] <a name="about-project"></a>
+
+
+**[# 📖 [My-Portfolio_Mobile-Version]  Project mainly focusing on building personal responsive portfolio page to showcase my skills and abiities and what I can offer.  
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+> Describe the tech stack and include only the relevant sections that apply to your project.
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>HTML5</li>
+  </ul>
+</details>
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li>CSS3</li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Displays_the_first_two_sections_of_the_overall_portfolio_project
+- Designed_to_be_responsive_and_mobile_friendly
+- Have_a_navigation_bar_section_and_hero_section
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+## 🚀 Live Demo <a name="live-demo"></a>
+
+>Not yet available
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+<!-- GETTING STARTED -->
+## 💻 Getting Started <a name="getting-started"></a>
+
+> Open the index.html file to view the contents 
+
+
+### Prerequisites
+
+In order to run this project you need:
+- Install Git in your computer [Git](https://git-scm.com/downloads)
+- Any text editor [VSCode](https://code.visualstudio.com/download) 
+- Install linters on your local env't
+ 
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder, access the directory and run the html file: 
+To clone: git clone git@github.com:Yosefgeda/My-portfolio.git
+to change to directory: cd My-portfolio
+Inside you will find the html file.
+
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
```

Comment:
[ Optional ] As a suggestion, this part should not be on your repo, because it was placed there on the TEMPLATE to guide you, but should not be seen for deployed or public repo like yours. please omit it